src/main/java/com/example/glovo/service/OrderService.java

```diff
@@ -0,0 +1,38 @@
+package com.example.glovo.service;
+
+
+import com.example.glovo.dto.OrderDto;
+import com.example.glovo.repository.OrderRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class OrderService {
+
+    private OrderRepository orderRepository;
+
+    public OrderDto getById(int id) {
+        return orderRepository.findById(id);
+    }
+
+    public boolean save(OrderDto orderDto) {
+        return orderRepository.save(orderDto);
+    }
+
+    public OrderDto update(int id, OrderDto orderDto) {
+        return orderRepository.update(id, orderDto);
```

Comment:
what if there is no the order with the id?

Reply:
nothing will be updated